setup.py

```diff
@@ -27,6 +27,7 @@ def read(fname):
     url="https://github.com/TimDettmers/bitsandbytes",
     packages=find_packages(),
     package_data={"": libs},
+    install_requires=['torch', 'numpy', 'scipy', 'pandas', 'matplotlib'],
```

Comment:
Are you absolutely sure `pandas` and `matplotlib` are _hard_ required for all use of the library? If not, they should be declared in extras, not here.